### PR TITLE
Add a generic shotgun to Aftershock

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Mods "Aftershock: Add a basic shotgun to the mod."

#### Purpose of change
#65562 Recently added shotguns using 00 ammo, but those are rather specialized firearms, and a standard, general use gun for that ammunition type was missing.

#### Describe the solution

Just a copy of the Mossberg 590  with new descriptions.

#### Testing
Loaded the game and fired the gun.

### CODE REVIEW

File: data/mods/Aftershock/items/gun/shot.json [hunk 1]
```diff
@@ -1,4 +1,25 @@
 [
+  {
+    "id": "afs_shotgun",
+    "copy-from": "mossberg_590",
+    "type": "GUN",
+    "name": { "str": "Benelli 315" },
+    "description": "A widely popular combat shotgun featuring minor iterative refinements to a base design nearly three centuries old.  Wether the design is outdated or unsurpassable remains a matter of personal opinion.",
```
Comment (TheShadowFerret) [inline data/mods/Aftershock/items/gun/shot.json:7]:
```suggestion
    "description": "A widely popular combat shotgun featuring minor iterative refinements to a base design nearly three centuries old.  Whether the design is outdated or unsurpassable remains a matter of personal opinion.",
```

File: data/mods/Aftershock/items/gun/shot.json [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+      {
+        "id": "vatforged",
+        "name": { "str": "vatforged shotgun" },
+        "description": "A crude but otherwise serviceable pump-action shotgun built from a hacked or scavenged template.  Frequently seen in the hands of frontier pirates, settlers or salvors.  Not that such groups are much different from one another.",
+        "weight": 3
+      },
+      {
+        "id": "military",
+        "name": { "str": "m210-UASTA shotgun" },
+        "description": "The standard issue combat shotgun of UICA marine and colonial forces.  Shot shells have poor performance against modern combat armor, so the M2101 is more often used during policing and peacekeeping operations.",
```
Comment (TheShadowFerret) [inline data/mods/Aftershock/items/gun/shot.json:18]:
```suggestion
        "name": { "str": "m2101-UASTA shotgun" },
        "description": "The standard issue combat shotgun of UICA marine and colonial forces.  Shot shells have poor performance against modern combat armor, so the M2101 is more often used during policing and peacekeeping operations.",
```
(Or change the `description` to be m210)

File: data/mods/Aftershock/items/gunmods/shotguns.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "afs_magnetic_choke",
+    "type": "GUNMOD",
+    "name": { "str": "magnetic choke" },
+    "//": "Yes lead isn't magnetic, no I don't care.",
```
Comment (TheShadowFerret) [inline data/mods/Aftershock/items/gunmods/shotguns.json:6]: ):<

Comment (John-Candlebury) [reply data/mods/Aftershock/items/gunmods/shotguns.json:6]: (:<

**Pull request merged**